ReSharper.FSharp/src/FSharp.Psi/src/CodeFormatter/FSharpFormatSettingsKey.cs

```diff
@@ -150,5 +150,8 @@ public class FSharpFormatSettingsKey : FormatSettingsKeyBase
 
     [SettingsEntry(true, "Never outdent pipe operators")]
     public bool NeverOutdentPipeOperators;
+
+    [SettingsIndexedEntry("List of Fantomas settings")]
+    public readonly IIndexedEntry<string, string> FantomasSettings;
```

Comment:
Why is it needed to be stored in R# settings?

Reply:
We only can read editorconfig via R# settings now. Don't think that creation of a separate EditorConfig reading API would simplify something.

Reply:
OK, I see, thanks.